github-actions/scan/src/input-helper.ts

```diff
@@ -21,6 +22,27 @@ export function split(input: string): string[] {
         .filter(item => item.length > 0);
 }
 
+export function ensureAcceptedScanType(data: string[]): string[] {
+    const accepted: string[] = [];
+    if (data){
+        for (const entry of data) {
+            if (equalIgnoreCase(entry, ScanType.CodeScan)) {
+                accepted.push(ScanType.CodeScan);
+            } else if (equalIgnoreCase(entry, ScanType.LicenseScan)) {
+                accepted.push(ScanType.LicenseScan);
+            } else if (equalIgnoreCase(entry, ScanType.SecretScan)) {
+                accepted.push(ScanType.SecretScan);
+            } else if (equalIgnoreCase(entry, ScanType.IacScan)){
+                accepted.push(ScanType.IacScan);
+            }
+        }
+    }
+    if (accepted.length == 0) {
```

Comment:
A comment that this is a fallback - for having at least code scan as accepted -would be nice